contracts/PrivateContract.sol

```diff
@@ -1,38 +1,72 @@
-//! Private Transactions State Storage
-//! By Parity Team, 2017.
-//!
-//! This contract keeps track of a private transaction state (supposedly encrypted)
-//! and allows its change only if all the Validators (from a static list, initialized in constructor)
-//! have signed a new state (hashed together with a current nonce, for replay protection).
+// Private Transactions State Storage
+//
+// This contract keeps track of a private transaction state (supposedly encrypted)
+// and allows its change only if all the Validators (from a static list, initialized in constructor)
+// have signed a new state (hashed together with a current nonce, for replay protection).
+//
+//
+// Copyright 2018 Parity Technologies Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 pragma solidity ^0.4.18;
 
 
 contract PrivateContract {
-    address[] public validators;
-    bytes public state;
-    bytes public code;
-    uint256 public nonce;
-
-    function PrivateContract(address[] initialValidators, bytes initialCode, bytes initialState) public {
-        validators = initialValidators;
-        code = initialCode;
-        state = initialState;
-        nonce = 1;
-    }
-    
-    function getValidators() public constant returns (address[]) {
-        return validators;
-    }
-
-    function setState(bytes newState, uint8[] v, bytes32[] r, bytes32[] s) public {
-        var noncedStateHash = keccak256([keccak256(newState), bytes32(nonce)]);
-
-        for (uint i = 0; i < validators.length; i++) {
-            assert(ecrecover(noncedStateHash, v[i], r[i], s[i]) == validators[i]);
-        }
-
-        state = newState;
-        nonce = nonce + 1;
-    }
+	address[] public validators;
+	bytes public state;
+	bytes public code;
+	uint256 public nonce;
+
+	function PrivateContract(
+		address[] initialValidators,
+		bytes initialCode,
+		bytes initialState
+	)
+		public
+	{
+		validators = initialValidators;
+		code = initialCode;
+		state = initialState;
+		nonce = 1;
+	}
+
+	function getValidators() public constant returns (address[]) {
```

Comment:
I would break the modifier list. I saw this code style in some other repo and I think it makes it easier to notice the visibility of a function. But I'm open to change if people don't like it.

Reply:
Do we have some accepted coding style guide for solidity contracts? As a result this contract looks like rust code )

Reply:
No I don't think we have any defined style guide but I think it would be good if we could have some consistency across our different contracts. I followed the style guide here https://solidity.readthedocs.io/en/v0.4.21/style-guide.html, some of that stuff is actually reported by the linter. And I added the rule of always breaking modifiers into new lines. Let me know if you think we should use a different style or add/remove some rules.

Reply:
Thanks for the link. It's very helpful (I'm not the experienced solidity coder). So it's important to have such guide fixed and not a big deal what's inside. I can bear anything :-)  As for breaking modifiers both options are ok.